src/components/FeaturedRepositories/FeaturedRepositories.tsx

```diff
@@ -1,95 +1,108 @@
-import React from 'react';
+import { Loader } from 'components/Loader';
+import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
+import IssueService from 'services/IssueService';
+import RepositoryService from 'services/RepositoryService';
+import { OrgRepo, Repository } from 'types/Repository';
 
 const maxWidth = 80;
-const projects = [
+const projects : Array<OrgRepo> = [
```

Comment:
Once projects are pulled from the database, I don't think it's necessary anymore to keep this static list. 
A list of of repositories that have votes would be fine. No need to combine these.

Reply:
it's removed in #34